lib/spack/spack/modules.py

```diff
@@ -327,6 +327,9 @@ def blacklisted(self):
         blacklist_matches = [x
                              for x in configuration.get('blacklist', [])
                              if self.spec.satisfies(x)]
+        blacklisted_as_implicit = (configuration.get('blacklist_implicits') and
```

Comment:
Why not writing something like:
```python
need_to_blacklist = configuration.get('blacklist_implicits')
installed_implicitly = not self.spec._installed_explicitly()
blacklisted_as_implicit = need_to_blacklist and installed_implicitly
```
and avoid the property below? I see no need to expose a new attribute in the class.